learning-development/learning-support.qmd

```diff
@@ -17,10 +17,36 @@ For best practice when using software and coding in our process, see our [guidan
 
 ## Technical workshops
 
- *	In person workshops covering specific technical skills in practice
- *	3 hours long, with people working in small groups
- *	Recently circled the sites running an introduction to Git, and introduction to R and RAP.
- * Contact statistics.development@education.gov.uk to register interest in workshops happening at your site or to request new topics!
+We currently offer two in person workshops covering specific technical skills in practice. Each workshop is around 3 hours long, and you will work in small groups. We travel to deliver workshops based on demand, so please get in touch and let us know if you'd like to join the waiting list in your area for one or both of the sessions. 
+
+Our current workshops are:
+
+**Introduction to Git and Azure DevOps**
+
+In this workshop, we cover:
+
+*	What are git, DevOps and GitHub and what are the differences?
+*	How can they help your work?
+*	How to start working with git and Dev Ops in the DfE ecosystem
+*	Managing projects and tasks with Dev Ops Boards
+*	Basic version control with git – add/stage, commit, push and pull
+*	Working with branches
+*	Tracking changes and who made them
+*	Merging, pull requests and merge conflicts
+
+**Introduction to R and RAP** 
+
+In this workshop, we cover:
+
+* Setting up an R project
```

Comment:
Do you know why the spacing is different for these bullet points compared to the ones above?

Reply:
I think this is a Github formatting issue because it looks ok in R Studio, but I'll check it again once it's in main 😄